### ISSUE TITLE
Document the connectors proto files

### ISSUE THREAD

**Issue by roee88** [mesh-for-data/mesh-for-data#252]
Currently many of the fields are undocumented and it's hard to understand what each field means. This is required before making changes as part of the work on taxonomy.

**Comment by roee88** [comment]
Do we want to close this as wontfix due to the expected changes as part of the taxonomy work? @rohithdv

**Comment by rohithdv** [comment]
> Do we want to close this as wontfix due to the expected changes as part of the taxonomy work? @rohithdv 

As the taxonomy work may change the protos, we could alternately  link this to taxonomy work and work on this issue once the taxonomy part is done. Please suggest. Thanks.

**Comment by roee88** [comment]
Closing in favor of REST based APIs (taxonomy related)
